lookout/style/format/rule_stat.py

```diff
@@ -52,7 +58,66 @@ def print_rule_table(rule_stat):
             new_line += col.ljust(max_cols[i])
             if i != len(line) - 1:
                 new_line += "|"
-        print(new_line)
+        res.append(new_line)
```

Comment:
StringIO passed as a file descriptor from the outside - see e.g. https://docs.python.org/3/library/pprint.html#pprint.pprint